java/src/main/java/org/rocksdb/WriteBufferManager.java

```diff
@@ -0,0 +1,30 @@
+package org.rocksdb;
+
+import org.rocksdb.Cache;
+
+/**
+ * Java wrapper over native write_buffer_manager class
+ */
+public class WriteBufferManager extends RocksObject {
```

Comment:
May be for later: others can contribute the remaining methods in native WriteBufferManager (like memroy_usage, FreeMem, etc).

Reply:
Yeah, maybe we will end up implementing it :)